Makefile

```diff
@@ -0,0 +1,43 @@
+PROJECT_PATH       = $(shell pwd)
```

Comment:
Ill just trust you on this one. I have never worked with makefiles.

Reply:
Oh, a bit foolish but this might be a good time to learn a little! I love to describe work flows with `make` even if it's not for compiling C/C++ applications. :)

If you find it hard to maintain you can always revert!

The makefile sets up a few variables. Everything thats assigned with `?=` will be overwritten if the macro is defined (see example in README), otherwise assigned with the right hand side of the expression. I then define the content of the shell script the make file will create. When running the target install the content will be printed to a file and the file will be made executable. When running uninstall it will remove the file. And yeah, since none of the targets are actual files on disk (that is, no file named `install` or `uninstall`), the targets are mared as `.PHONY`.